scripts/shared/prefsSetsDbUtils.js

```diff
@@ -0,0 +1,80 @@
+/*!
+Copyright 2018 Raising the Floor US
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+/**
+ * This file contains utilities that are useful to generate keys data that
+ * can be inserted into a CouchDB.
+ *
+ * Actually, these utils are used in:
+ *   * convertPrefs.js
+ *   * loadEmptyKeys.js
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.registerNamespace("gpii.prefsSetsDbUtils");
+
+/**
+ * This is an empty preferences set.
+ */
+gpii.prefsSetsDbUtils.emptyPreferencesBlock = {
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {}
+            }
+        }
+    }
+};
+
+/**
+ * Generate the data for a key creation.
+ *
+ * The data contains both the gpiiKey and the preferences safe.
+ *
+ * @param  {String} gpiiKeyId - The identifier of the prefsSet (usually, a uuid).
+ * @param  {Object} preferences - The preferences to include in the prefsSet.
```

Comment:
This `preferences` block can be made optional. If not provided, use `gpii.prefsSetsDbUtils.emptyPreferencesBlock` as the default. Add this line into the function:
```
var preferences = preferences || gpii.prefsSetsDbUtils.emptyPreferencesBlock;
```
So that the caller script doesn't need to be aware of the existence of `gpii.prefsSetsDbUtils.emptyPreferencesBlock`.